ca/config_test.go

```diff
@@ -211,7 +211,7 @@ func testGRPCConnection(t *testing.T, secConfig *ca.SecurityConfig) {
 	}
 	conn, err := grpc.Dial(l.Addr().String(), dialOpts...)
 	require.NoError(t, err)
-	conn.Close()
+	_ = conn.Close()
```

Comment:
The error-returns from `conn.Close()` are checked in other tests in this file. What makes this instance different?

Reply:
Nothing different. I guess I started with ignoring (as many repos tend to), but found quite some other unhandled errors.